app/helpers/application_table_helper.rb

```diff
@@ -85,9 +81,9 @@ def remove_access_link(application, user = nil)
 
     link_to(
       path,
-      class: "govuk-button govuk-button--warning govuk-!-margin-0",
+      class: "govuk-button govuk-button--warning govuk-!-margin-0 applications-table__remove_access_link",
       data: { module: "govuk-button" },
-    ) { button_or_link_content("Remove access", "to", application.name) }
+    ) { button_or_link_content("Remove", "access to", application.name) }
```

Comment:
Maybe out of scope for this PR but I'd find it helpful if `button_or_link_content` took keyword arguments, would make it a little easier to read at a glance